src/main/java/cz/it4i/parallel/ui/HPCSettingsScreenWindow.java

```diff
@@ -0,0 +1,92 @@
+
+package cz.it4i.parallel.ui;
+
+
+
+import cz.it4i.parallel.runners.HPCSettings;
+import javafx.scene.Scene;
+import javafx.stage.Modality;
+import javafx.stage.Stage;
+import javafx.stage.Window;
+
+public class HPCSettingsScreenWindow 
+{
+
+	private HPCSettingsScreenController controller;
+
+	private HPCSettings settings;
```

Comment:
Settings previously used for same window(in case that profile does not contain settings) should be probably stored with PrefService.

Reply:
Settings used for the same window are now stored with PrefService using the helper class "LastFormLoader".